pyasic/config/__init__.py

```diff
@@ -92,6 +101,7 @@ def as_inno(self, user_suffix: str = None) -> dict:
         }
 
     def as_bosminer(self, user_suffix: str = None) -> dict:
+        """Generates the configuration in the format suitable for Bosminer."""
```

Comment:
This gets put into a file on the device, but is for the same type of device as the Boser function, just slightly older.  I would prefer to have this be `Generates the configuration in the bosminer.toml format.`